src/posts/2020-06-08-going-remote.md

```diff
@@ -0,0 +1,110 @@
+
+There's nothing wrong with dailies per se, but you should be aware - **they are a tool for project productivity, not for combating isolation**. This happened during a time when most of the team was in-office and all the goofing and joking around happened within it. Jokes wouldn't be on Slack. Nobody even would say anything on #random and if I took the initiative, I would barely get someone replying and would feel ignored and isolated. And afterwards I'd see everyone happy on Twitter. This is daunting. This is scary. And it leads to a negative feedback loop because after you neglect them, they will feel isolated and reduce their interactions, leading them to be even more neglected by other coworkers and increasing their isolation. **Applications like Slack, Basecamp or whatever, are the main method your coworkers have to communicate with you. Don't leave them hanging**. That's your tool to combat isolation, not dailies. Dailies are work, not watercooler, beer sipping, chess bashing fun times.
+
+(Yeah and those Friday references - we allocate our Friday to what we call "investment time". Improving ourselves and our company, learning new languages, frameworks, techniques, trying out weird things, giving talks to each other and so on. More on that later.)
+
+Another powerful tool for battling isolation is to **create actual moments for goofing around**. Every week, at 3 PM, we have a 30 min call to discuss how our previous week was. It's not about work, it's about our personal lives, just to share what we've been doing, how Pedro's house is *still* not finished even though he's been claiming "just this one thing missing" for the past year and a half, how the new Cardi B single is 🔥🔥🔥, you know, the *yoozh*. Every Friday, at the end of the day we have a call to discuss what we accomplished in our investment time. This tends to include people trying out fancy new beers and sharing them. Every day, at the end of the day, there's an opt-in call for folks to just chat. And occasionally you see people on Slack creating calls while they are just coding or doing their *thing*, each on their own project, but all in the same call to mimic the office environment (beware this might hurt your productivity if you need to be focused and should be handled with care). Create and nurture those moments, **it's your team, they're your co-workers** and sometimes we all need to talk about chess and the Sicilian defence being overrated.
```

Comment:
> Every week, at 3 PM, we have a 30 min call

You state a very specific hour but not the day it happens on. Would make more sense to state the day and be fuzzy about the hour. :stuck_out_tongue: 

> how Pedro's house is *still* not finished

:triumph: